### DISCUSSION TITLE
[Bug] `scoop checkup` giving error

### DISCUSSION THREAD

**Discussion by RayyanNafees** [ScoopInstaller/Scoop#5667] [Help and Questions]
#### Current Behavior

Running `scoop checkup` results in error in the console

#### Expected Behavior

Should give information regarding how to fix the issues just like `flutter doctor` does

### System details

**Windows version:** 11

**OS architecture:**  64bit

**PowerShell version:** 
![image](https://github.com/ScoopInstaller/Scoop/assets/59685062/8261c6c7-6dbf-4d62-8b17-07cc14ac42dd)

#### Scoop Configuration

```json
{
  "last_update": "2023-09-28T12:28:59.9165064+05:30",
  "scoop_branch": "master",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by HUMORCE** [comment]
Can you switch real-time protection status manually?
